components/bibtemplate/src/auro-bibtemplate.js

```diff
@@ -9,8 +9,8 @@
 import { LitElement } from "lit";
 import { html } from "lit/static-html.js";
 
-import styleCss from "./styles/style-css.js";
-import colorCss from "./styles/color-css.js";
+import tokenCss from "./styles/style-css.js";
+import colorCss from "./styles/tokens-css.js";
```

Comment:
I think you need to keep this part how it was. You should be importing all 3 files.